src/main/java/beam/analysis/plots/PersonTravelTimeAnalysis.java

```diff
@@ -139,13 +159,35 @@ public void createRootGraphForAverageCarTravelTime(IterationEndsEvent event) thr
         createCarAverageTimesGraphForRootIteration(averageCarDatasetForRootIteration,carMode,fileName);
     }
 
-    Tuple<List<String>, Tuple<double[][], Double>> compute() {
+    private void createRootGraphForAverageBusTravelTime(IterationEndsEvent event) throws IOException{
+        double[][] singleBusDataSet = new double[1][event.getIteration()+1];
+        for (int i =0 ;i <= event.getIteration() ;i++){
+            System.out.println("Bus mode times : " + busAverageTime.get(i));
+            singleBusDataSet[0][i] = busAverageTime.get(i);
+        }
+        CategoryDataset averageBusDatasetForRootIteration = buildAverageTimeDatasetGraphForRoot(busMode,singleBusDataSet);
+        OutputDirectoryHierarchy outputDirectoryHierarchy = event.getServices().getControlerIO();
+        String fileName = outputDirectoryHierarchy.getOutputFilename( averageBusTravelTimeRootGraphName + ".png");
+        createCarAverageTimesGraphForRootIteration(averageBusDatasetForRootIteration,busMode,fileName);
+    }
+
+    private void createRootGraphForAverageCavTravelTime(IterationEndsEvent event) throws IOException{
+        double[][] singleCavDataSet = new double[1][event.getIteration()+1];
+        for (int i =0 ;i <= event.getIteration() ;i++){
+            System.out.println("CAV mode times : " + cavAverageTime.get(i));
```

Comment:
don't use system out

Reply:
Opps .. sorry I added it while debugging , will remove it